webapps-schema/src/main/java/io/camunda/webapps/schema/entities/operate/post/PostImporterQueueEntity.java

```diff
@@ -5,13 +5,13 @@
  * Licensed under the Camunda License 1.0. You may not use this file
  * except in compliance with the Camunda License 1.0.
  */
-package io.camunda.operate.entities.post;
+package io.camunda.webapps.schema.entities.operate.post;
 
-import io.camunda.operate.entities.OperateEntity;
+import io.camunda.webapps.schema.entities.AbstractExporterEntity;
 import java.time.OffsetDateTime;
 import java.util.Objects;
 
-public class PostImporterQueueEntity extends OperateEntity<PostImporterQueueEntity> {
+public class PostImporterQueueEntity extends AbstractExporterEntity<PostImporterQueueEntity> {
```

Comment:
:question: Are post importer actions still relevant with the new exporter? If not, should we leave it in the operate module itself?

Reply:
It is still relevant and we can't leave `PostImporterQueueEntity` in Operate as they are created base on Incident Zeebe records.